### PR TITLE
Create a layout for the homepage

### PR DESCRIPTION
**Pull request by kashiish**
These commits create a basic homepage which includes a search box to search for a topic. I have added an HTML, CSS, and JS file for the page. I also added the SearchServlet that takes care of getting search results. Right now, this servlet just returns a static response.

### CODE REVIEW

File: src/main/webapp/homepage.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Sullivan</title>
+    <link rel="stylesheet" type="text/css" href="homepage.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
```
Comment (betrevisan) [inline src/main/webapp/homepage.html:7]: What does this line do?

File: src/main/webapp/homepage.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  </head>
+  <body>
+    <div id="content">
+        <div id="search-form">
+            <input type="text" id="topic-search-box" placeholder="What do you need help with today?">
```
Comment (betrevisan) [inline src/main/webapp/homepage.html:12]: I like this prompt :)

File: src/main/java/com/google/sps/servlets/SearchServlet.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** Servlet that returns comments from and adds comments to Datastore. */
```
Comment (betrevisan) [inline src/main/java/com/google/sps/servlets/SearchServlet.java:25]: I think we should change this comment

Comment (kashiish) [reply src/main/java/com/google/sps/servlets/SearchServlet.java]: Oh ya, my bad! Just changed it.

File: src/main/webapp/homepage.html (hunk 1 above)
Comment (kashiish) [reply src/main/webapp/homepage.html:7]: I think it's for mobile browser support.

File: src/main/java/com/google/sps/servlets/SearchServlet.java [hunk 4]
```diff
@@ -22,7 +22,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 
-/** Servlet that returns comments from and adds comments to Datastore. */
+/** Servlet that returns search results (tutors and books) for a topic. */
```
Comment (betrevisan) [inline src/main/java/com/google/sps/servlets/SearchServlet.java:25]: Ignore my previous comment on this

File: src/main/webapp/homepage.html (hunk 1 above)
Comment (betrevisan) [reply src/main/webapp/homepage.html:7]: Cool! I'll add that to mine too then :)

**Comment by betrevisan** [comment]
Also, make sure to add a copyright notice to the top of css and html files.

**Pull request merged**